SleepEngine/Engine/GameTemplate/Game.h

```diff
@@ -71,3 +74,5 @@ class Game
 };
 
 END_NAMESPACE_SLEEP
+
+#include "global_shortcuts.h"
```

Comment:
Why here?

Reply:
the shortcuts (almost) always get member of game, so i thought it would be better to include it just after the def of game

Reply:
On my opinion a bit confusing, but ok